src/main/java/core/basesyntax/service/OperationHandlerServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.HandlerStorage;
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.strategy.OperationHandler;
+
+public class OperationHandlerServiceImpl implements OperationHandlerService {
+    public void changeAmount(FruitRecord record) {
+        char symbol = record.getOperation()
+                .name()
+                .toLowerCase()
+                .charAt(0);
+        OperationHandler handler = HandlerStorage.getHandlerByLetter(symbol);
```

Comment:
could we store `FruitRecord.Operation` as key in HandlerMap()? Then we don't need to get operation first letter every time

Reply:
fixed